app/include/zmk/events/keycode-state-changed.h

```diff
@@ -14,16 +14,20 @@ struct keycode_state_changed {
     u8_t usage_page;
     u32_t keycode;
     bool state;
+    u32_t position;
```

Comment:
These events might be synthesized from other place, not just from a specific keybinding, per-se. Why was this needed?

Reply:
we can do without.